diff -urN gnome-bluetooth-0.7.0.orig/python/manager.py gnome-bluetooth-0.7.0/python/manager.py
--- gnome-bluetooth-0.7.0.orig/python/manager.py	2005-08-07 18:51:50.000000000 +0200
+++ gnome-bluetooth-0.7.0/python/manager.py	2006-04-03 04:39:08.000000000 +0200
@@ -265,7 +265,7 @@
         dialog.show ()
 
     def image_file (self, fname):
-        for d in [os.path.join (__datadir__, "pixmaps"), "../pixmaps"]:
+	for d in [os.path.join (__datadir__, "pixmaps"), os.path.join (__datadir__, "../pixmaps"), "../pixmaps"]:
             if os.path.isfile (os.path.join (d, fname)):
                 return os.path.join (d, fname)
         return None
