--- gnome-bluetooth-0.6.0/src/fileactiondialog.gob.wiget	2005-06-22 15:44:19.000000000 +0000
+++ gnome-bluetooth-0.6.0/src/fileactiondialog.gob	2005-08-21 10:04:22.000000000 +0000
@@ -32,6 +32,10 @@
 #include <libgnomevfs/gnome-vfs-utils.h>
 #include <gnome.h>
 
+#ifndef GNOME_ICON_LOOKUP_FLAGS_ALLOW_SVG_AS_THEMSELVES
+#define GNOME_ICON_LOOKUP_FLAGS_ALLOW_SVG_AS_THEMSELVES 1<<2
+#endif
+
 #define GNOMEBT_FILEACTION_CLOSE 0x01
 #define GNOMEBT_FILEACTION_OPEN 0x02
 #define GNOMEBT_FILEACTION_DELETE 0x03
