--- gnome-bluetooth-0.5.1/obex/gnome-obex-server.desktop.in~	2005-05-23 15:07:39.000000000 +0200
+++ gnome-bluetooth-0.5.1/obex/gnome-obex-server.desktop.in	2005-05-23 15:07:39.000000000 +0200
@@ -4,9 +4,9 @@
 _Comment=Receive files sent by Bluetooth devices
 Icon=../gnome-bluetooth/pixmaps/blueradio-48.png
 Exec=gnome-obex-server
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;System;
+Categories=System;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-bluetooth
 X-GNOME-Bugzilla-Component=obex-server
