diff -Nur gnome-bluetooth-0.5.orig/src/Makefile.am gnome-bluetooth-0.5/src/Makefile.am
--- gnome-bluetooth-0.5.orig/src/Makefile.am	2004-06-04 20:05:29.000000000 +0200
+++ gnome-bluetooth-0.5/src/Makefile.am	2004-06-12 03:39:13.000000000 +0200
@@ -85,10 +85,10 @@
 regenerate-defs: gnomebt-controller.defs gnomebt-chooser.defs
 
 gnomebt-controller.defs: gnomebt-controller.h
-	python /usr/share/pygtk/2.0/codegen/h2def.py -v gnomebt-controller.h > gnomebt-controller.defs
+	python /usr/share/pygtk/2.0/codegen/h2def.pyc -v gnomebt-controller.h > gnomebt-controller.defs
 
 gnomebt-chooser.defs: gnomebt-chooser.h
-	python /usr/share/pygtk/2.0/codegen/h2def.py -v gnomebt-chooser.h > gnomebt-chooser.defs
+	python /usr/share/pygtk/2.0/codegen/h2def.pyc -v gnomebt-chooser.h > gnomebt-chooser.defs
 
 gnomebt-controller-py.c: gnomebt-controller.defs gnomebt-controller.override
 	pygtk-codegen-2.0 --prefix controller \
