--- obex/gnome-obex-send.c
+++ obex/gnome-obex-send.c
@@ -81,7 +81,7 @@
 			OBEX_OBJPUSH_SVCLASS_ID);
 
 	for (item=list; item != NULL; item = g_slist_next(item)) {
-		ret = (uint8_t)((guint) item->data);
+		ret = GPOINTER_TO_UINT(item->data);
 	}
 
 	g_object_unref (btmanager);
--- src/controller.gob
+++ src/controller.gob
@@ -464,7 +464,7 @@
 
         for (item=chanlist; item != NULL && num < 0; item=g_slist_next(item)) {
             num = gnomebt_controller_get_rfcomm_port(self, bdaddr,
-                (guint)item->data);
+                GPOINTER_TO_UINT(item->data));
         }
         g_slist_free(chanlist);
         return num;
@@ -491,12 +491,12 @@
             bdaddr, service);
         for (item=chanlist; item != NULL && num < 0; item=g_slist_next(item)) {
             num = gnomebt_controller_get_rfcomm_port (self, bdaddr,
-                (guint)item->data);
+                GPOINTER_TO_UINT(item->data));
             if (num < 0) {
                 num=gnomebt_controller_connect_rfcomm_port (self,
-                    bdaddr, (guint)item->data );
+                    bdaddr, GPOINTER_TO_UINT(item->data) );
                  g_message("rfcomm attempting to connect %s chan %d, got %d",
-                    bdaddr, (guint)item->data,num);
+                    bdaddr, GPOINTER_TO_UINT(item->data),num);
             }
         }
         g_slist_free(chanlist);
--- src/fileactiondialog.gob
+++ src/fileactiondialog.gob
@@ -30,6 +30,8 @@
 #include <gtk/gtk.h>
 #include <libgnomevfs/gnome-vfs-mime-handlers.h>
 #include <libgnomevfs/gnome-vfs-utils.h>
+#include <libgnomevfs/gnome-vfs-init.h>
+#include <libgnomevfs/gnome-vfs-ops.h>
 #include <gnome.h>
 
 #define GNOMEBT_FILEACTION_SAVE 0x01
--- src/gnomebt-controller-test.c
+++ src/gnomebt-controller-test.c
@@ -129,7 +129,7 @@
         GnomebtServiceDesc *desc = (GnomebtServiceDesc *)item->data;
         printf ("   Service ID: %x\n", desc->id);
         for (chans=desc->channels; chans!=NULL; chans = g_slist_next (chans)) {
-            printf ("      Channel: %d\n", (guint) chans->data);
+            printf ("      Channel: %d\n", GPOINTER_TO_UINT(chans->data));
         }
     }
     gnomebt_service_list_free (services);
@@ -137,7 +137,7 @@
     chans = gnomebt_controller_channels_for_service (bc, TBDADDR, 0x1101);
     printf ("Channels for Service ID: %x\n", 0x1101);
     for ( ; chans!=NULL; chans = g_slist_next (chans)) {
-        printf ("  Channel: %d\n", (guint) chans->data);
+        printf ("  Channel: %d\n", GPOINTER_TO_UINT(chans->data));
     }
 
     printf ("Known devices:\n");
--- src/gnomebt-controller.override
+++ src/gnomebt-controller.override
@@ -90,7 +90,7 @@
         PyObject *chanlist = PyList_New (num_s);
         for (chans=desc->channels, j=0; chans; chans=g_slist_next(chans), ++j) {
             PyList_SET_ITEM(chanlist, j,
-                Py_BuildValue("i", (guint)chans->data));
+                Py_BuildValue("i", GPOINTER_TO_UINT(chans->data)));
         }
         PyDict_SetItem(ret, Py_BuildValue("i", desc->id), chanlist);
     }
@@ -122,7 +122,7 @@
     ret = PyList_New (len);
     for (l = slist, i = 0; l; l = g_slist_next (l), ++i) {
         PyList_SET_ITEM(ret, i,
-            Py_BuildValue("i", (guint)l->data));
+            Py_BuildValue("i", GPOINTER_TO_UINT(l->data)));
     }
     g_slist_free(slist);
     return ret;
